x/bank/spec/02_keepers.md

```diff
@@ -59,6 +59,7 @@ The base keeper provides full-permission access: the ability to arbitrary modify
 // between accounts.
 type Keeper interface {
     SendKeeper
+    WithMintCoinsRestriction(NewRestrictionFn BankMintingRestrictionFn) BaseKeeper 
```

Comment:
These needs more of a spec update, perhaps a new subsection, below ## Blocklisting Addresses